draft-ietf-dprive-dnsoquic.md

```diff
@@ -552,31 +552,27 @@ connections. For example:
 
 ## Flow Control Mechanisms
 
-Servers and Clients manage flow control as specified in QUIC.
-
-Servers can use the "maximum stream ID" parameter of the QUIC transport to limit
-the number of streams opened by the client. This mechanism will effectively
-limit the number of DNS queries that a client can send on a single DoQ
-connection. The initial value of this parameter is specified by the transport
-parameter `initial_max_streams_bidi`. To allow at least one query per client,
-servers SHOULD use a non-zero value for this parameter. Using larger values
-will result in better performance.
-
-The flow control mechanisms of QUIC control how much data can be sent by QUIC
-nodes at a given time. The initial values of per stream flow control parameters
-is defined by two transport parameters:
-
-* initial_max_stream_data_bidi_local: when set by the client, specifies the
-  amount of data that servers can send on a "response" stream without waiting
-  for a MAX_STREAM_DATA frame.
-
-* initial_max_stream_data_bidi_remote: when set by the server, specifies the
-  amount of data that clients can send on a "query" stream without waiting for
-  a MAX_STREAM_DATA frame.
-
-For better performance, it is RECOMMENDED that clients and servers set each of
-these two parameters to a value of 65537 or greater, allowing for at least one
-query or response of maximum length 65535 bytes.
+Servers and Clients manage flow control using the mechanisms defined in
+section 4 of {{!RFC9000}}. These mechanisms allows clients and servers to
```

Comment:
```suggestion
{{Section 4 of RFC9000}}. These mechanisms allows clients and servers to
```
(Gonna keep doing that to you :)

Reply:
I tried. I updated to the latest release of kramdown, using `gem update`. Then I applied your edit, ran kramdown, and ran xml2rfc. I get the following error:
```
PS C:\Users\huite\GitHub\dnsoquic> xml2rfc --text draft.xml
draft.xml(0): Error: IDREF attribute target references an unknown ID "Section_4_of_RFC9000", at None
file:/C:/Users/huite/GitHub/dnsoquic/draft.xml(13): Error: Invalid document before running preptool.
Unable to complete processing draft.xml
```
I think I will table these `section` updates for now.